starknet_py/net/full_node_client.py

```diff
@@ -46,7 +46,7 @@
 from starknet_py.net.models.transaction import (
     AccountTransaction,
     Declare,
-    DeclareSchema,
+    DeclareV1Schema,
```

Comment:
Sorry, but it gets confusing, why do we add v1 to declare schema and not deploy account and invoke?